cube3d.html

```diff
@@ -1381,191 +1384,193 @@ <h4>Utilisation de textures</h4>
 							Si la hauteur de notre portion de mur est supérieure à la hauteur de la texture, cela veut dire qu'il faudra étirer la texture en hauteur pour pouvoir remplir toute la zone.<br>
-							A l'inverse, si notre portion est moins haute, il faudra "écraser" la texture pour que celle-ci puisse être entièrement affichée.
+							À l'inverse, si notre portion est moins haute, il faudra "écraser" la texture pour qu'elle puisse être entièrement affichée.
 						</p>
 
 						<div class="row">
 							<div class="col-4">
 								<span class="image fit">
 									<img src="images/cube3d/stretch0.png" alt="" />
-									<center><sup><i>Texture de base</i></sup></center>
+									<center><sup><i>texture de base</i></sup></center>
 								</span>
 							</div>
 							<div class="col-4">
 								<span class="image fit">
 									<img src="images/cube3d/stretch1.png" alt="" />
-									<center><sup><i>Texture étirée</i></sup></center>
+									<center><sup><i>texture étirée</i></sup></center>
 								</span>
 							</div>
 							<div class="col-4">
 								<span class="image fit">
 									<img src="images/cube3d/stretch2.png" alt="" />
-									<center><sup><i>Texture écrasée</i></sup></center>
+									<center><sup><i>texture écrasée</i></sup></center>
 								</span>
 							</div>
 						</div>
 
-						<p>Bien sûr ce changement d'échelle dépendra de l'orientation du joueur par rapport au mur. C'est d'ailleurs celui-ci qui va rendre l'incrustation de la texture naturelle.</p>
-
+						<p>
+						  Bien sûr ce changement d'échelle dépendra de l'orientation du joueur par rapport au mur<!-- tu pourrais détailler plus j'ai pas trop capté -->.
+						  C'est d'ailleurs ce qui va rendre l'incrustation de la texture naturelle.</p>
 						<div class="row">
 							<div class="col-6">
 								<span class="image fit">
 									<img src="images/cube3d/no_texture.png" alt="" />
-									<center><sup><i>Mur sans texture</i></sup></center>
+									<center><sup><i>mur sans texture</i></sup></center>
 								</span>
 							</div>
 							<div class="col-6">
 								<span class="image fit">
 									<img src="images/cube3d/texture.png" alt="" />
-									<center><sup><i>Mur avec texture</i></sup></center>
+									<center><sup><i>mur avec texture</i></sup></center>
 								</span>
 							</div>
 						</div>
 
 						<p>
-							Pour trouver le pas permettant d'afficher à l'échelle une texture, il suffit de diviser la hauteur de la texture par la hauteur de notre portion de mur.<br>
-							Il ne reste alors qu'à partir de haut en bas la portion de mur et d'y déposer les pixels de la texture.
+							Pour trouver le ratio permettant d'afficher à l'échelle une texture, il suffit de diviser la hauteur de la texture par la hauteur de notre portion de mur.<!-- Du coup tu mets à l'échelle en hauteur mais c'est comment en largeur ? --><br>
```

Comment:
Dans le cas où la largeur de la bande vertical est de 1px, il suffit de simplement poser les pixels à la suite.
Quand la bande est plus large que 1px il faut placer le même pixel sur toute la largeur de la bande.
Concrètement dans le code c'est une boucle for (ligne 1573 de ce fichier)

C'est peut être pas très clair, on pourra en discuter au besoin

+ Commentaire à supprimer 
+ Eclaircir ce point dans la phrase ?